bin/oxfordshire/send-rdi-emails

```diff
@@ -40,6 +40,10 @@ foreach my $inspector (@inspectors) {
         user => $inspector,
         mark_as_processed => 1,
     };
+    my $email_params = {
+        %$params,
+        staging_site => FixMyStreet->config('STAGING_SITE'),
```

Comment:
If we're having separate email params, only the dates and user are shared, don't really need mark_as_processed or inspection_date here, so could be split more. But no biggie.